sources/Application/Model/Project.cpp

```diff
@@ -27,6 +28,11 @@ tempoNudge_(0)
 	this->Insert(wrap) ;
 	Variable *transpose=new Variable("transpose",VAR_TRANSPOSE,0) ;
 	this->Insert(transpose) ;
+    
+    // from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Model/Project.cpp
```

Comment:
I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.